index.js

```diff
@@ -213,6 +213,8 @@ function genResponse (response) {
 
   Object.keys(response).forEach(key => {
     if (response[key].type) {
+      if (response[key].type === 'null') delete response[key].type
```

Comment:
can you duplicate the object if you plan to make edits?

Reply:
Not sure what your mean. If you edit your code to provide an object, you get the result of the object in the swagger documentation which is also valid for swagger.

Reply:
I can add a unit test, but we because we are dealing with the swagger UI, we need nightmare to verify results we are getting in the documentation. I can go ahead and do this if you wish.

Reply:
If you are good with the change, I'll commit a piece to the README concerning empty responses as well.

Reply:
As this caused a crash when the route was accessed, just add a test that does not crash the server/throw exception when rendering.

As for creating a copy: just create a new jnstance with the property you need if you want to delete things.

If you need only a one-level copy you can use Object.assign({}, obj) (check this), or if you need a deep copy use the clone npm module.

Reply:
kk